### ISSUE TITLE
NPM Audit: Prototype pollution in async

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy#2327]
* `jake` dependency (filed at https://github.com/jakejs/jake/issues/408) via `ejs`
  * alternatively https://github.com/mde/ejs/issues/659
* `portscanner` dependency (no issue yet https://github.com/baalexander/node-portscanner) via `browser-sync`

```
async  <2.6.4
Severity: high
Prototype Pollution in async - https://github.com/advisories/GHSA-fwr7-v2mv-hh25
fix available via `npm audit fix --force`
Will install @11ty/eleventy@0.3.3, which is a breaking change
node_modules/async
node_modules/portscanner/node_modules/async
  jake  >=8.0.1
  Depends on vulnerable versions of async
  node_modules/jake
    ejs  >=3.1.2
    Depends on vulnerable versions of jake
    node_modules/ejs
      @11ty/eleventy  >=0.3.4
      Depends on vulnerable versions of browser-sync
      Depends on vulnerable versions of ejs
      node_modules/@11ty/eleventy
  portscanner  0.1.1 - 2.1.1
  Depends on vulnerable versions of async
  node_modules/portscanner
    browser-sync  <=0.6.2 || >=2.0.0-rc1
    Depends on vulnerable versions of portscanner
    node_modules/browser-sync
```

**Comment by zachleat** [comment]
Applies to both 1.x and 2.x branches, although I expect that this will be fixed upstream in a minor dependency bump.

**Comment by sentience** [comment]
Looks like `portscanner@2.2.0` already [allows the fix, `async@2.6.4`](https://github.com/baalexander/node-portscanner/blob/master/package.json#L35).

`browser-sync` [specifies exactly `portscanner@2.1.1`](https://github.com/BrowserSync/browser-sync/blob/master/packages/browser-sync/package.json#L56), so I guess that's where an issue/fix is needed.

**Comment by SphinxKnight** [comment]
Matching issue on `browser-sync` repo: https://github.com/BrowserSync/browser-sync/issues/1959

**Comment by shakyShane** [comment]
released as browser-sync@2.27.10 - thank you :)

**Comment by zachleat** [comment]
```
~/Temp/eleventy-install-may-18-2022 ᐅ npm install @11ty/eleventy           

added 322 packages, and audited 323 packages in 7s

26 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

Verified, thank you @shakyShane!
